src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultItems.props

```diff
@@ -1,12 +1,20 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup>
-    <EnableDefaultAndroidItems Condition=" '$(EnableDefaultAndroidItems)' == '' ">$(EnableDefaultItems)</EnableDefaultAndroidItems>
-  </PropertyGroup>
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.DefaultItems.props
+WARNING:  This file is imported by AutoImport.props, and will be imported by all
+          projects using Microsoft.NET.Sdk.  All Item includes in this file *MUST* be
+          hidden behind an Android property based condition.
+
+Default item includes (globs and implicit references).
+
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <!-- Default Resource file inclusion -->
   <!-- https://developer.android.com/guide/topics/resources/providing-resources -->
-  <ItemGroup Condition=" '$(EnableDefaultAndroidItems)' == 'true' ">
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == 'Android' and '$(EnableDefaultAndroidItems)' == 'true' ">
```

Comment:
@dsplaisted Is this `$(TargetPlatformIdentifier)` check restrictive enough for these default items being imported by `AutoImport.props`?

Reply:
I think in this case you don't need the `TargetPlatformIdentifier` check, because `EnableDefaultAndroidItems` will only be true if when targeting Android anyway.